narwhals/series.py

```diff
@@ -1582,23 +1582,23 @@ def sort(self, *, descending: bool = False, nulls_last: bool = False) -> Self:
 
             We define library agnostic functions:
 
-            >>> def my_library_agnostic_function(s_native: IntoSeriesT) -> IntoSeriesT:
+            >>> def agnostic_sort(s_native: IntoSeriesT) -> IntoSeriesT:
             ...     s = nw.from_native(s_native, series_only=True)
             ...     return s.sort().to_native()
 
-            >>> @nw.narwhalify
-            ... def func_descend(s):
+            >>> def agnostic_sort_descending(s_native: IntoSeriesT) -> IntoSeriesT:
+            ...     s = nw.from_native(s_native, series_only=True)
             ...     return s.sort(descending=True).to_native()
 
-            We can then pass either pandas or Polars to `func`:
+            We can then pass either pandas or Polars to `agnostic_sort`:
 
-            >>> my_library_agnostic_function(s_pd)
+            >>> agnostic_sort(s_pd)
```

Comment:
🤔 ok i do like the way this looks...nice one @FBruzzesi

Reply:
Thanks! I like it as well! Drawback is... a bit more effort 😂

Reply:
sprint 😎